README.rst

```diff
@@ -146,13 +146,14 @@ equivalent without it. For example:
 
 * Rewrite ``list([f(x) for x in foo])`` as ``[f(x) for x in foo]``
 
-C412: Unnecessary list comprehension - 'in' can take a generator.
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+C412: Unnecessary (dict/list/set) comprehension - 'in' can take a generator.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-It's unnecessary to pass a list comprehension to 'in' that can take a
+It's unnecessary to pass a dict/list/set comprehension to 'in' that can take a
 generator instead. For example:
 
 * Rewrite ``y in [f(x) for x in foo]`` as ``y in (f(x) for x in foo)``
+* Rewrite ``y in {x ** 2 for x in foo}`` as ``y in (f(x) for x in foo)``
```

Comment:
The rewritten version doesn't relate to the original.

Reply:
Fixed in #216.